site/public/js/admin-gradeable-updates.js

```diff
@@ -276,8 +313,10 @@ function ajaxCheckBuildStatus() {
         }
     });
 }
-function setRandomGraders(gradeable_id,p_values,successCallback,errorCallback) {
-    var number_to_grade=$('#number_to_peer_grade').val();
+function setRandomGraders(gradeable_id,p_values,successCallback,errorCallback,all_grade_all) {
+    var number_to_grade=1;
+    if(all_grade_all===true){number_to_grade=10000;}
```

Comment:
lets make these `if else` statements go in their own line e.g:

`if (all_grad_all === true){
   number_to_grade = 10000;
}else{
...
}

Reply:
Done